blog/2019-11-27-webService.md

```diff
@@ -0,0 +1,53 @@
+```
+id: webService
+title: 什么是Web Service
+author: Robbie Han
+author_title: Front End Engineer @ Tradeshift
+author_url: https://github.com/USTC-Han
+author_image_url: https://robbie-pic.oss-cn-beijing.aliyuncs.com/IMG_4175.JPG?x-oss-process=style/compress
+tags: [webService]
+```
+
+## 什么是Web Service：
+
+​		相比于Web Service（网络服务），“本地服务”这个名词我们似乎更加的熟悉。本地服务通过本地的计算机环境为系统功能提供服务，完成某项特定的功能，使用期间不需要使用网络。而对应的Web Service则是利用网络调用其他网站的资源来实现系统的某项特定功能。
+
+​		例如，当我们想在个人网站中添加一个显示天气的功能时，如果我们仅仅使用一些简单的前端组件显然是无法实现的。这项功能的数据可能不仅依赖于数据库的分析，甚至还需要卫星的探测等。此外，天气展示这项功能并不是我们这个网站中的核心功能，我们不会为了实现这项功能付出太多的开发成本。
+
+​	   于是乎，Web Service就应运而生了。对于像在网页上显示天气、地图、Twitter上的最新动态这类的非核心功能，我们不需要自己去开发，Web Service就可以让我们的网站使用其他网站的资源。**这些网络资源会向外界暴露出能够通过Web进行调用的API，我们只需要调用这些API，就可以拿到这些网络资源的数据**。
+
+<!--truncate-->
```

Comment:
What does this do?

Reply:
if we remove '<!--truncate-->' , the blog will show all content in the home page. Now, it will only show the content before it.

Reply:
got it